spec/queries/orders/filtered_state_query_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe Orders::FilteredStateQuery do
+  let(:params_all) { 'all' }
+  let(:params_in_queue) { 'in_queue' }
+  let(:params_in_delivery) { 'in_delivery' }
+  let(:params_delivered) { 'delivered' }
+  let(:params_canceled) { 'canceled' }
+  let(:orders) { Order.where(id: create_list(:order, 5).map(&:id), state: 'in_delivery') }
+
+  describe '#filtered_orders' do
```

Comment:
please test logic inside this query object, currently you check if result is ActiveRecord::Relation, it's wrong